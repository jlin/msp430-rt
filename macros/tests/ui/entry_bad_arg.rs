#![no_main]

use msp430_rt_macros::entry;

#[entry(arg)]
fn main() -> ! {
    loop {}
}
