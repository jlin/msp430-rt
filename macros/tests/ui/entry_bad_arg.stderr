error: expected `interrupt_enable` or no arguments at all
 --> tests/ui/entry_bad_arg.rs:5:9
  |
5 | #[entry(arg)]
  |         ^^^
